modules/ROOT/pages/planning-and-tuning/operators/operators-detail.adoc

```diff
@@ -311,6 +311,49 @@ Total database accesses: 2, total allocated memory: 184
 ======
 
 
+[[query-plan-directed-relationship-by-element-id-seek]]
+== Directed Relationship By Element Id Seek
+// DirectedRelationshipByElementIdSeek
+
+The `DirectedRelationshipByElementIdSeek` operator reads one or more relationships by element id from the relationship store, and produces both the relationship and the nodes on either side.
```

Comment:
What is meant by "on either side" here? Do you mean "and the nodes on either side of the relationship" or something else?

```suggestion
The `DirectedRelationshipByElementIdSeek` operator reads one or more relationships by element id from the relationship store (specified via the function xref::functions/scalar.adoc#functions-elementid[elementId()]) and produces both the relationship and the nodes on either side.
```